packages/core/src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -42,6 +43,7 @@ export interface RemoveIndexQueryOptions {
 }
 
 export const REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS = new Set<keyof RemoveIndexQueryOptions>(['concurrently', 'ifExists', 'cascade']);
+export const TRUNCATE_TABLE_SUPPORTABLE_OPTIONS = new Set<keyof TruncateTableOptions>(['cascade', 'restartIdentity']);
```

Comment:
```suggestion
export const TRUNCATE_TABLE_QUERY_SUPPORTABLE_OPTIONS = new Set<keyof TruncateTableQueryOptions>(['cascade', 'restartIdentity']);
```

See also my other suggestions relating to this

Reply:
Done